flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/TwoPhaseCommitSinkFunction.java

```diff
@@ -304,4 +287,33 @@ private String name() {
 			getRuntimeContext().getIndexOfThisSubtask(),
 			getRuntimeContext().getNumberOfParallelSubtasks());
 	}
+
+	/**
+	 * State POJO class coupling pendingTransaction, context and pendingCommitTransactions.
```

Comment:
This is not technically a Flink POJO because there is no zero-argument constructor.

Reply:
Added default constructor.